app/views/publisher_information/beta_capabilities.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-third">
+    <ul class="govuk-list">
+      <li>
+        <%= link_to t("publisher_information.beta_capabilities.title"), beta_capabilities_path, class: "govuk-link" %>
+      </li>
+      <li>
+        <%= link_to t("publisher_information.how_to.title"), how_to_path, class: "govuk-link" %>
+      </li>
+      <li>
+        <%= link_to t("publisher_information.index.title"), publisher_updates_path, class: "govuk-link" %>
+      </li>
+    </ul>
+  </div>
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class='govuk-heading-m'><%= t("publisher_information.beta_capabilities.title") %></h1>
+    <%= render "govuk_publishing_components/components/govspeak" do %>
+        <%= govspeak_to_html File.read("app/views/publisher_information/beta_capabilities.govspeak.md") %>
+    <% end %>
+    <%= render 'request_feature' %>
```

Comment:
Convention would be to use `<%= render partial: "request_feature" %>`

Reply:
I no longer have 'request_feature' as a partial view.  But I did change all references to the 'publication_list_list' partial view to include that.